packages/nimble-components/src/dialog/specs/README.md

```diff
@@ -78,35 +80,74 @@ We will not make any special effort to provide forms support (i.e. form `type="d
+The visual design spec has a few different layouts for the dialog. Not all layouts will be supported in the initial styling pass of the dialog. See below for details on what parts will and will not be supported in the intial styling pass:
+
+-   Title
+    -   Included in initial styling pass: Yes
+    -   Rationale: The title is a fundamental part of the dialog. Adding the title will also resolve an accessibility issue with the dialog's aria-label.
+    -   Additional details: The content provided for the title will be used as the label of the dialog for accessibility purposes.
+-   Subtitle
+    -   Included in initial styling pass: Yes
+    -   Rationale: The styling of the subtitle is straight-forward and there are no open questions associated with it.
+-   Warning message
+    -   Included in initial styling pass: No
+    -   Rationale: There isn't a pressing need for this part of the dialog yet, and there are a number of questions that need to be resolved prior to adding the messages:
+        -   Should the messages be arbitrary content provided by the user, or should we have an API around providing message(s) with a specified severity, icon, color, etc?
+        -   What, if any, is the overlap between these messages and a future `<nimble-banner>` component?
+        -   Do we allow multiple messages at the same time? If so, is there a maximum height the messages can consume within the dialog?
+-   Close button
+    -   Included in initial styling pass: No
+    -   Rationale: There isn't a pressing need for this component yet, and there are a number of questions that need to be resolved prior to adding the button:
+        -   Should the API allow a client to slot any arbitrary buttons, including a close button, or should we only support one single button that closes the dialog?
+        -   What control do we need to give over the close button? Do we need a way for the user to explicitly disable it?
+        -   Should there be any connection between the `prevent-dismiss` attribute on the dialog and the visibility or disabled state of the close button?
+-   Content
+    -   Included in initial styling pass: Yes
+    -   Rationale: This is critical for using a dialog
+-   Footer buttons
+    -   Included in initial styling pass: Yes
+    -   Rationale: These buttons are required for interacting with the dialog.
+    -   Additional details:
+        -   The footer container will have a `flex` layout to allow a client to easily align buttons in the appropriate place within the footer. There will not be explicit slots within the dialog for the various alignments of buttons in the footer.
```

Comment:
Could you clarify what layout we expect the dialog will do and what layout clients can override? e.g. 
1. Will the dialog put all the buttons on the right by default? 
2. If so, will it allow a button to be placed on the left (like the Back button in the design spec)? 
3. Will it provide standard horizontal padding?
4. Will it ensure OK buttons are always to the right of Cancel buttons?

(I think I know the answer to some of these, just want to make it explicit)